docs/docs/advanced/air-gap.md

```diff
@@ -22,6 +22,15 @@ Download `db.tar.gz`:
     $ oras pull -a ghcr.io/aquasecurity/trivy-db:2
     ```
 
+=== "Trivy"
```

Comment:
How about the order? Trivy should take precedence over oras.
Also, the oras installation should be inside the block. Otherwise, users will try to download oras even though they want to use `--download-db-only`.

Reply:
Done